starter/src/main/java/com/vivy/springdoc/OpenAPIAutoConfiguration.java

```diff
@@ -32,4 +50,52 @@ public OpenAPI bearerOpenAPI() {
                 ))
                 .addSecurityItem(new SecurityRequirement().addList("bearer-jwt", List.of("read", "write")));
     }
+
+    @Bean
+    public ApplicationRunner applicationRunner(AppConfiguration configProperties) {
+
+        return new ApplicationRunner() {
+            @Override
+            public void run(ApplicationArguments args) throws Exception {
+                ResponseEntity<String> apiDocsResponse = getRestTemplate().getForEntity(String.format("http://localhost:%d/v3/api-docs", configProperties.getPort()), String.class);
+                if (apiDocsResponse.getStatusCode() == HttpStatus.OK) {
+                    String output = configProperties.output;
```

Comment:
```suggestion
                    String output = configProperties.getOutput();
```

minor codestyle - we don't use fields directly